services/fxa/serverless.yml

```diff
@@ -150,6 +155,9 @@ functions:
           method: ANY
           path: '{proxy+}'
           cors: true
+      - sns:
+          topicName: fxa-data
```

Comment:
any reason why this isnt just `fxa-event-data` to match more closely the Display Name?

Reply:
changed to ${self:provider.stage}-fxa-data as we are defining the SNS Topics as such and this is how FxA is subscribing.